Sources/StreamChatSwiftUI/Resources/Assets.xcassets/Attachments/Previews/aac.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "aac.pdf",
```

Comment:
`aac.pdf`?

Reply:
yes, most of the icons are PDFs, for easier support of multiple resolutions (same as in UIKit)